core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -714,7 +715,8 @@ class MultiCloudWorkspaceService(override val ctx: RawlsRequestContext,
       createdDate = currentDate,
       lastModified = currentDate,
       createdBy = ctx.userInfo.userEmail.value,
-      attributes = attributes
+      attributes = attributes,
+      WorkspaceState.Ready
```

Comment:
A future ticket will change this to be `Creating`.

Reply:
@aherbst-broad actually, I realized we don't have a ticket for `Creating`. Even though creation will remain synchronous for the time being, I think we may want a ticket to make sure that we put the Workspace state to `CreateFailed` if we encounter a failure (in either GCP or Azure workspaces) and are unable to completely roll back and delete the Workspace record. Thoughts?